specifyweb/frontend/js_src/lib/components/SchemaConfig/Table.tsx

```diff
@@ -44,59 +48,77 @@ export function SchemaConfigTable({
+          label: schemaText.table(),
+          value: container.name,
+        })}
+      >
+        <Label.Block>
+          {schemaText.caption()}
+          <Input.Text
+            isReadOnly={isReadOnly || name === undefined}
+            maxLength={maxSchemaValueLength}
+            required
+            value={name?.text ?? ''}
+            onValueChange={(text): void => handleChangeName({ ...name!, text })}
+          />
+        </Label.Block>
+        <Label.Block>
+          {schemaText.description()}
+          <AutoGrowTextArea
+            className="resize-y"
+            isReadOnly={isReadOnly || desc === undefined}
+            maxLength={maxSchemaValueLength}
+            value={desc?.text ?? ''}
+            onValueChange={(text): void => handleChangeDesc({ ...desc!, text })}
+          />
+        </Label.Block>
+        <FormatterPicker
+          container={container}
+          schemaData={schemaData}
+          type="format"
+          onChange={(format): void => handleChange({ ...container, format })}
         />
-      </Label.Block>
-      <Label.Block>
-        {schemaText.description()}
-        <AutoGrowTextArea
-          className="resize-y"
-          isReadOnly={isReadOnly || desc === undefined}
-          maxLength={maxSchemaValueLength}
-          value={desc?.text ?? ''}
-          onValueChange={(text): void => handleChangeDesc({ ...desc!, text })}
-        />
-      </Label.Block>
-      <FormatterPicker
-        container={container}
-        schemaData={schemaData}
-        type="format"
-        onChange={(format): void => handleChange({ ...container, format })}
-      />
-      <FormatterPicker
-        container={container}
-        schemaData={schemaData}
-        type="aggregator"
-        onChange={(aggregator): void =>
-          handleChange({ ...container, aggregator })
-        }
-      />
-      <Label.Inline>
-        <Input.Checkbox
-          checked={container.isHidden}
-          isReadOnly={isReadOnly}
-          onValueChange={(isHidden): void =>
-            handleChange({ ...container, isHidden })
+        <FormatterPicker
+          container={container}
+          schemaData={schemaData}
+          type="aggregator"
+          onChange={(aggregator): void =>
+            handleChange({ ...container, aggregator })
           }
         />
-        {schemaText.hideTable()}
-      </Label.Inline>
-    </SchemaConfigColumn>
+        <Label.Block>
+          <Button.Small onClick={(): void => setUniquenessOpen()}>
+            {schemaText.uniquenessRules()}
+          </Button.Small>
+        </Label.Block>
+        <Label.Inline>
+          <Input.Checkbox
+            checked={container.isHidden}
+            isReadOnly={isReadOnly}
+            onValueChange={(isHidden): void =>
+              handleChange({ ...container, isHidden })
+            }
+          />
+          {schemaText.hideTable()}
+        </Label.Inline>
+      </SchemaConfigColumn>
+      {isUniquenessOpem && (
```

Comment:
I had initially proposed the same thing. However, it was decided it would be more valuable to instead see all of a table's uniqueness rules at once rather than going through the individual fields in the schema config and only seeing the rules to that field. 

@specify/ux-testing Is this still the consensus/decision?